src/main/java/fr/lunatech/timekeeper/services/EventTemplateService.java

```diff
@@ -68,20 +68,30 @@
     @Transactional
     public Optional<Long> create(EventTemplateRequest request, AuthenticationContext ctx) {
         logger.debug("Create a new event template with {}, {}", request, ctx);
-        final EventTemplate eventTemplate = request.unbind(userService::findById, ctx);
+        final EventTemplate eventTemplate = request.unbind(ctx);
+
+        eventTemplate.isValid();
+
         // by unbinding we also generate userEvent in db for each user
         // user event attributes will be inherited from eventTemplate (startTime, etc.)
-        boolean checkEvent = validateEvent(
+        checkThatNoOtherEventSameNameSameDateAlreadyExists(
                 eventTemplate.name,
                 eventTemplate.id,
                 eventTemplate.startDateTime,
                 eventTemplate.endDateTime, ctx);
 
-        if (checkEvent) {
-            return Optional.empty();
-        }
-
         eventTemplate.persistAndFlush();
+
+        // Create a userEvent for each attendee using the userEventService
+        if (!request.getAttendees().isEmpty()) {
```

Comment:
If attendees are absent, then are we skipping "Creation" of new Event ?

Reply:
it's userEvent (event that are specific for each user). So if you send an EventTemplate with no attendess, then it cannot create any userEvent. `request` here is an EventTemplateRequest. 
Here is a sample : let's say you want to create a Devoxx event, for a specific date, but you don't know yet who will attend this conference => you can create an EventTemplateRequest with an empty list of attendees (no one selected in the GUI)